### PR TITLE
ChimeraStation.

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

This PR adds the ChimeraStation

Lore: 
Several stations have been destroyed by the syndicate and others forces, causing NT to have financial losses. NT has started the "Chimera" project, this project aims to use parts of the stations that have been destroyed, repair these parts and unite them in a single station, with this NT tries to reduce their losses to a minimum.

       MAP:
MAP

      AREAS:
AREAS

    =  THINGS TO DO: =

- [x]  1. Science
- [x]  2. Cargo 
- [X]  3. Medbay
- [x]  4. Brigde 
- [x]  5. AI Chambers 
- [x]  6. secretary zone 
- [x]  7.  Departures  
- [x] 8.  Arrivals (IS UGLY THo)
- [x] 9. Chapel 
- [x] 10. Library
- [x] 11. More+ maint
- [x] 12. Cry (all + cry + cry in maint + cry in apc)
- [x] 13. Finish sec
- [x] 14. Stuff in maint (god help me)
- [x] 15. Disposals
- [x] 16. void tech zone
- [x] 17. Cameras in the hallway

## Why It's Good For The Game

More maps mean less routine

## Changelog
:cl:
add: Added map
add: clown and mime office areas.
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
The Error is an easy fix, Annie, don't feel to bad about it!

**Comment by DaddyIssues98** [comment]
4 of 6 checks, this is a victory royale

**Comment by DaddyIssues98** [comment]
i need to see how to fix the checks.

**Comment by DaddyIssues98** [comment]
thanks @Kitsunemitsu  for the fix!

**Comment by DaddyIssues98** [comment]
FOR FUCK SAKE NOW THERE IS A ANOTHER ERROR

**Comment by DaddyIssues98** [comment]
is fixed, dont know how BUT DEAR GOD IS FUCKING FIXED

File: code/game/area/space_station_13_areas.dm [hunk 1]
```diff
@@ -1036,6 +1045,16 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	airlock_wires = /datum/wires/airlock/service
 	sound_environment = SOUND_AREA_STANDARD_STATION
 
+/area/office/clown
```
Comment (JohnFulpWillard) [inline code/game/area/space_station_13_areas.dm:1048]: Are these necessary?

**Comment by JohnFulpWillard** [comment]
There's suppsoed to have a .json file for things like Chef's CQC

File: code/game/area/space_station_13_areas.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply code/game/area/space_station_13_areas.dm:1048]: It's just a goofy thing for Chimera, I think it's funny

**Pull request merged**